source/runbooks/oracle-licence-discovery.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+owner_slack: "#modernisation-platform"
+title: oracle licence discovery
+last_reviewed_on: 2024-12-05
+review_in: 6 months
+---
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-NXTCMQ7ZX6"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-NXTCMQ7ZX6');
+</script>
+
+# <%= current_page.data.title %>
+
+# Centrally Track Oracle Database Licenses in AWS Organizations
+
+## Introduction
+
+AWS License Manager and AWS Systems Manager can be used together to centrally track Oracle database licenses across an AWS Organization. This approach helps ensure compliance and optimize license usage.
```

Comment:
Licence vs License

You are using both in this, which could be correct, but I think its licence in British English vs USA. Similar to Organizations vs Organisations. However as its AWS product its American

Reply:
changed to british english